adopt-a-dog/styles/styles.css

```diff
@@ -0,0 +1,117 @@
+.container {
+  padding-left: 7%;
+  padding-right: 7%;
+  padding-top: 2%;
+  padding-bottom: 2%;
+}
+/* .clearfix::after {
+  content: "";
+  display: block;
+  clear:both;
+} */
+h1{
+  font-family: 'Francois One', sans-serif;
+  font-style: italic;
+  color: white;
+}
+h2{
+  font-family: 'Jacques Francois', serif;
+  color: blue;
+}
+h3{
+  font-family: 'Jacques Francois', serif;
+  color: blue;
+  font-size: 1.1rem;
+}
+h4{
+  font-family: 'Francois One', sans-serif;
+  color: white;
+  font-size: .8rem;
+}
+p{
+  font-family: 'Francois One', sans-serif;
+  font-size: .8rem;
+  font-weight: 100;
+}
+a{
+  font-family: 'Francois One', sans-serif;
+  font-size: 1rem;
+}
+nav {
+  overflow: auto;
+  background-color: silver;
+  color: blue;
+}
+nav li {
+  display: flex;
+  float: left;
+  margin: 0;
+  padding: .5rem 1rem .5rem 0;
+}
+nav ul {
+  list-style-type: none;
+  padding: 0;
+}
+.header{
+  height: 300px;
+  box-sizing: border-box;
+  width: auto;
+  background-color: darkblue;
+  background-size: auto;
+  background-repeat: no-repeat;
+  align-content: center;
+}
+.header-cart {
```

Comment:
Be careful with vertical-align. It's not very consistent. I'd suggest looking into position absolute. You'll need to put a position relative on the parent container. Hit up the w3 schools page on position (I actually prefer it to the MDN docs) or this page is somewhat helpful http://learnlayout.com/position.html